command/issues/list/list.go

```diff
@@ -135,3 +166,73 @@ func (opts *IssuesListOptions) showIssues() {
 	// Using pterm to render the list of list
 	pterm.DefaultTable.WithSeparator("\t").WithData(opts.ptermTable).Render()
 }
+
+// exportJSON exports issues as JSON
+func (opts *IssuesListOptions) exportJSON(filename string) (err error) {
+	var issueJSON ExportData
+	issueJSON, err = convertJSON(opts.issuesData)
+	if err != nil {
+		return err
+	}
+
+	data, err := json.MarshalIndent(issueJSON, "", "	")
+	if err != nil {
+		return err
+	}
+
+	if filename == "" {
+		pterm.Println(string(data))
+	} else {
+		err = ioutil.WriteFile(filename, data, 0644)
+		if err != nil {
+			return err
+		}
+
+		pterm.Info.Printf("Saved issues to %s!\n", filename)
+	}
```

Comment:
We can write this in a cleaner way using guard clause:
```go
func exportJSON(...) err error {
  ...
  if filename == "" {
		pterm.Println(string(data))
		return nil
  }

  if err = ioutil.WriteFile(filename, data, 0644); err != nil {
			return err
  }
  pterm.Info.Printf("Saved issues to %s!\n", filename)
  return nil
}
```

Also, are we handling the case in which the user doesn't explicitly specify a filename for the JSON file? In that case we should export the issues to `deepsource_issues.json` file.

Reply:
Cobra doesn't allow blank filenames when `--output-file` is passed. So, the user needs to pass a value while specifying the `--output-file` flag.

Reply:
Cool, what error does it display when the filename is not passed?

Reply:
`ERROR` flag needs an argument: --output-file